packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx

```diff
@@ -18,58 +18,57 @@ const PlatformBox = ({ platform: { icon, title, description } }) => (
         </div>
     </>
 );
-class PlatformDropdown extends React.PureComponent {
-    handleClickOutside = event => {
+
+// eslint-disable-next-line react/display-name
```

Comment:
Why we are disabling this? 
is it possible to add the displayName property on the component?